innersource-license.md

```diff
@@ -0,0 +1,94 @@
+- There are legal contracts in place which cover the sharing of source code. These contracts are not standardized, so they create additional effort in negotiating and understanding for every project. The existing contracts may also not allow sharing source code in an open enough sense to support a true InnerSource approach.
+- Alternatively, there are no legal contracts in place but source code is shared informally. That might create uncertainty in cases where clarity about ownership and rights and obligations is needed.
+
+## Solutions
+
+Creating an **InnerSource License** customized to the needs of the organization in question (and their legal entities). This license needs to be generic enough to be applied to the most important inter-company relationships.
+
+It is important to write the InnerSource License such that it truly allows for OpenSource-like collaborations across the boundaries of the involved legal entities. Therefore the 4 freedoms of free software should be integrated into the license.
+
+The License is written as a formal legal document, and can be used as part of contracts between the legal entities to govern the code sharing agreements.
+
+## Resulting Context
+
+With the InnerSource License, we have a tool to share code between legal entities within our organization.
+
+The license simplifies the conversations within our organization about sharing source code, and is motivating the first legal entities to do so.
+
+**Note:** The experiment described in **Known Instances** is in an early phase. Therefore a firm **Resulting Context** has not formed yet. In a couple of months the effects of the InnerSource License on this problem space will be more clear, and this section can be updated.
+
+## Known Instances
+
+DB Systel created their own InnerSource License, see [DB Inner Source License][db-inner-source-license]. They used the [EUPL][eupl], as that offered an open-source-like starting point, and then worked out the constraints and additional rules required in their specific organizational context.
+
+The first legal entities (companies) within the DB AG are using their InnerSource License.
+
+One positive effect that is already showing is that it simplifies the conversation, especially if some of the involved parties don’t know the InnerSource concept that well yet. Licenses are a well-known concept, therefore having an InnerSource License is a great discussion starter.
+
+The experiments are also uncovering that there are further collaboration challenges that need to be solved in order to lead to a true InnerSource contribution and collaboration model.
+
+The mentioned collaboration challenges include:
+
+- making InnerSource licensed projects discoverable
+- building communities for collaboration on projects, just like in Open Source
+
+It is worth mentioning that so far the software shared under this InnerSource license is mostly tooling, infrastructure, and tools lower in the stack.
+
```

Comment:
Add Bosch as an known instance, too. Here's my proposal for content:

"The InnerSource initiative at Robert Bosch GmbH was also governed with a license from the beginning in 2009. With over 500 legal entities, sharing software internally was very difficult and time consuming to implement in a compliant way. InnerSource and specifically the InnerSource license changed that completely. Today, collaboration in software projects governed by their InnerSource license is the de facto standard and widely practiced. Bosch has evolved their license over the years, e. g. to also allow the eventual publication of InnerSource software as Open Source software, which wasn't possible with early versions of their license."